.circleci/config.yml

```diff
@@ -124,6 +128,42 @@ jobs:
       - store_test_results:
           path: /tmp/artifacts/code-coverage/
 
+  i18n:
+    docker:
+      - image: circleci/node:latest-browsers-legacy
+    steps:
+      - checkout
+      - run:
+          name: Update npm
+          command: sudo npm install -g npm@latest
+      - run:
+          name: Install SVN
+          command: sudo apt-get update && sudo apt-get install subversion
+      - run:
+          name: Install PHP
+          command: sudo apt-get install php libapache2-mod-php php-mbstring
+      - run:
+          name: Install gettext
+          command: sudo apt-get install gettext
+      - run:
+          name: Install node packages
+          command: npm install
+      - run:
+          name: Build the plugin
+          command: |
+            mkdir -p /tmp/artifacts
+            npm run babel:build
+            npm run pot2json
+            npm run json2po
+            npm run po2mo
+            npm run po2jed
```

Comment:
@EvanHerman These lines can be removed. The babel:build script already does makepot which calls pot2json, and the other scripts are for dev use only. Basically all this job has to do is ensure two files are updated with every master commit: coblocks.pot and coblocks.json.